<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteSourceGUI</class>
 <widget class="RollupWidget" name="RemoteSourceGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Remote source</string>
  </property>
  <widget class="QWidget" name="settingsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="dataAddressLayout">
      <item>
       <widget class="QLabel" name="dataAddressLabel">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataAddress">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Local data listener address</string>
        </property>
        <property name="inputMask">
         <string>000.000.000.000</string>
        </property>
        <property name="text">
         <string>0...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="dataPortSeparator">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dataPort">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Local data listener port</string>
        </property>
        <property name="inputMask">
         <string>00000</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Set local data listener address and port</string>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="nominalValuesLayout">
      <item>
       <widget class="QLabel" name="sampleRateLabel">
        <property name="text">
         <string> SR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sampleRate">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Stream nominal sample rate</string>
        </property>
        <property name="text">
         <string>0000000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sampleRateUnits">
        <property name="text">
         <string>S/s</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="monitorLayout">
      <item>
       <widget class="QLabel" name="nominalNbBlocksText">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Nb total blocks / Nb FEC blocks</string>
        </property>
        <property name="text">
         <string>000/00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="allFramesDecoded">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Frames status: green = all original received, none = some recovered by FEC, red = some lost, blue = remote not streaming</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../sdrgui/resources/res.qrc">
          <normaloff>:/locked.png</normaloff>:/locked.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="streamRateText">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Stream actual sample rate</string>
        </property>
        <property name="text">
         <string>0000000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="streamRateUnits">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>S/s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eventCountsReset">
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="eventUnrecText">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Number of unrecoverable errors since event counts reset</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="eventRecText">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Number of correctable errors since event counts reset</string>
        </property>
        <property name="text">
         <string>000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="eventCountsTimeText">
        <property name="toolTip">
         <string>Time since last event counts reset</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="queueLengthLayout">
      <item>
       <widget class="QLabel" name="queueLengthLabel">
        <property name="text">
         <string>QL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="queueLengthGauge">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>14</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Queue length gauge</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="queueLengthText">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Queued data blocks / Queue size in data blocks</string>
        </property>
        <property name="text">
         <string>000/000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollupWidget</class>
   <extends>QWidget</extends>
   <header>gui/rollupwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
